make/platform/Makefile.cross-aarch64

```diff
@@ -0,0 +1,18 @@
+# Copyright 2004-2016 Cray Inc.
+# Other additional copyright holders may be indicated within.
+# 
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# 
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include $(CHPL_MAKE_HOME)/make/platform/Makefile.cray-x-series
```

Comment:
I don't think you should include this Makefile here - it's only meant for Cray X-series systems.  If this really needs anything that is set in Makefile.cray-x-series it should be set directly here instead.  I doubt anything in there is needed here though.